src/app/chart/donut/donut.component.html

```diff
@@ -0,0 +1 @@
+<div #chartElement id="{{donutChartId}}"></div>
```

Comment:
We will need to create the pf-empty-chart for the no data available use case. This is an integral part of the component HTML layout in Angular Patternfly. It should be a very simple component -- please let me know if you need help with that?

Reply:
I presume this is another issue and PR? I can take a look at porting this. I suppose it's:

https://github.com/patternfly/angular-patternfly/blob/master/src/charts/empty-chart.component.js
https://github.com/patternfly/angular-patternfly/blob/master/src/charts/empty-chart.html